src/ingest_validation_tools/docs_utils.py

```diff
@@ -469,4 +487,24 @@ def _make_dir_description(dir_schema):
         row.append(description_md)
 
         output.append('| ' + ' | '.join(row) + ' |')
-    return '\n'.join(output)
+    table = '\n'.join(output)
+    pipeline_infos_md = ' and '.join(make_pipeline_link(info) for info in pipeline_infos)
+    pipeline_blurb = \
```

Comment:
So we want this written out with or without a pipeline part of the docs for an assay?

Reply:
It seems we only have one of these pipeline info blurbs

Reply:
> So we want this written out with or without a pipeline part of the docs for an assay?

No: The `pipeline_blurb` will be be empty if there are no pipelines. See below: `if pipeline_infos else ''`

> It seems we only have one of these pipeline info blurbs

Yes, for now: Didn't want to invest too much here when it's not clear that this has buy-in.